lib/rtree.ml

```diff
@@ -11,41 +11,41 @@
 open Util
 
 (* Type of regular trees:
-   - Param denotes tree variables (like de Bruijn indices)
+   - Var denotes tree variables (like de Bruijn indices)
      the first int is the depth of the occurrence, and the second int
      is the index in the array of trees introduced at that depth.
-     Warning: Param's indices both start at 0!
+     Warning: Var's indices both start at 0!
    - Node denotes the usual tree node, labelled with 'a
    - Rec(j,v1..vn) introduces infinite tree. It denotes
      v(j+1) with parameters 0..n-1 replaced by
      Rec(0,v1..vn)..Rec(n-1,v1..vn) respectively.
  *)
```

Comment:
This doc should be updated to explain the implicit DNF semantics of the Node constructor.

Reply:
Doc updated.